posthog/migrations/0362_auto_20231107_2008.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 3.2.19 on 2023-11-07 20:08
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("posthog", "0361_add_plugin_config_ui_fields"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="team",
+            name="external_data_workspace_last_synced",
+            field=models.DateTimeField(blank=True, null=True),
+        ),
+        migrations.AddField(
+            model_name="team",
+            name="external_data_workspace_rows_synced_in_month",
+            field=models.IntegerField(blank=True, null=True),
```

Comment:
big int to prevent future headaches? I wouldn't be surprised if someone synced > 2 billion rows in a month

(when I don't know the costs lol)